### ISSUE TITLE
Warning: Lint.Fix is not a constructor

### ISSUE THREAD

**Issue by lumio** [palantir/tslint#2575]
### Bug Report

- __TSLint version__: 5.1.0
- __TypeScript version__: 2.2.2
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
// Linting works fine
// No code is affected

// Running ./node_modules/.bin/tslint -t stylish `find . -type f -name '*.tsx'`
```

with `tslint.json` configuration:
```json
{
    "defaultSeverity": "error",
    "extends": [
      "tslint:latest", "tslint-react", "tslint-eslint-rules"
    ],
    "jsRules": {},
    "rules": {
      "align": false,
      "ordered-imports": false,
      "jsx-no-multiline-js": false,
      "quotemark": [ true, "single", "avoid-escape", "jsx-single" ],
      "jsx-boolean-value": [ "never" ],
      "typedef-whitespace": [
         true,
         {
            "call-signature": "onespace",
            "index-signature": "onespace",
            "parameter": "onespace",
            "property-declaration": "onespace",
            "variable-declaration": "onespace"
         }
      ],
      "arrow-parens": [ true, "ban-single-arg-parens" ],
      "object-literal-sort-keys": false,
      "no-use-before-declare": true,
      "trailing-comma": [
         true,
         {
            "multiline": "always",
            "singleline": "never"
         }
      ],
      "jsx-no-lambda": false,
      "one-line": false,
      "variable-name": [ true, "ban-keywords", "check-format", "allow-leading-underscore", "allow-pascal-case" ],

      "space-in-parens": [ true, "always" ],
      "array-bracket-spacing": [ true, "always" ],
      "brace-style": [ true, "stroustrup" ]
   },
    "rulesDirectory": []
}
```

#### Actual behavior
Receiving the following warning:
`Warning: Lint.Fix is not a constructor`

#### Expected behavior
No warning at all

**Comment by andy-hanson** [comment]
Tslint by default catches all errors and just prints the message, which isn't very helpful.
You could go into `node_modules/tslint/lib/linter.js` and comment out the `catch` block in `applyRule` (along with the `try` block wrapper) to get the error stacktrace.

**Comment by mmckenziedev** [comment]
I'm having the same issue. `tslint-react@2.6.0` depends on `tslint@4`. I solved the issue by downgrading from `tslint@5` to `tslint@4`. It looks like there have been API changes which `tslint-react` needs to accommodate.

**Comment by lumio** [comment]
Thanks to both of you. So indeed @andy-hanson, tslint-eslint-rules caused some problems, but only on Linux systems.
The real solution was downgrading to tslint@4.5.1 @mmckenziedev 

To my surprise it enabled further rules which I am really thankful for! I didn't realize that tslint-react was not working proberly 😅

**Comment by adidahiya** [comment]
I just released [tslint-react 3.0.0](https://github.com/palantir/tslint-react/releases/tag/3.0.0), which is compatible with tslint 5+
